tidy3d/components/eme/grid.py

```diff
@@ -249,6 +319,26 @@ def make_grid(self, center: Coordinate, size: Size, axis: Axis) -> EMEGrid:
             mode_specs=self.mode_specs,
         )
 
+    @classmethod
+    def from_structures(
+        cls, structures: List[Structure], axis: Axis, mode_spec: EMEModeSpec, **kwargs
+    ) -> EMEExplicitGrid:
+        """Create an explicit EME grid with boundaries aligned with
+        structure bounding boxes."""
```

Comment:
Since this is user-facing it should have a proper doc string with parameters and return, and maybe an example.